Islandora-CLAW/carapace
# Moving to at_core 8.x-3.1

# What does this Pull Request do?

This pull request updates Carapace so that it uses the latest (as of August 2018) base theme provided by Adaptivetheme (8.x-3.1)

# What's new?

Carapace has been updated so that users can make more changes to the site's look and feel using the interface provided by the theme settings. These changes include:

- Additional Menu settings available using the Responsive Menus extension tab.
- Additional fonts available using the Fonts extension tab.
- Additional Touch Icons for mobile devices available using the Touch Icons extension tab.
- Additional Shortcodes available using the Shortcodes extension tab.
- Automated creation of Flexslider Slideshows using the Slideshows extension tab.
- Additional Regions available for display and configuration using the Layouts tab.
- Additional default color schemes using the Color Scheme tab. (Known issue: Some colors are not mapping to the correct regions.)

# How should this be tested?

Make sure you have the 8.x-3.1 of Adaptivetheme (https://www.drupal.org/project/adaptivetheme) version installed in your /themes folder. Also, make sure you have installed the 8.x-3.2 version of at_tools (https://www.drupal.org/project/at_tools) in your /modules folder. Clone or download this version of Carapace to /themes/custom and enable as you would enable any other theme. More info at: http://docs.adaptivethemes.com/

# Additional Notes:

This Pull Request required the creation of a new branch since it is incompatible with the previous version of Carapace, which was created using the 8.x-2.0 base.

# Interested parties
@Islandora-CLAW/committers
